src/test/java/juja/microservices/users/entity/UserTest.java

```diff
@@ -2,21 +2,23 @@
 
 import org.junit.Test;
 
+import java.util.UUID;
+
 import static org.junit.Assert.*;
 
 public class UserTest {
 
     @Test
     public void shouldReturnUser() {
-        String expectedUuid = "uuid";
+        UUID expectedUuid = UUID.fromString("00000000-0000-0001-0000-000000000002");
         String expectedFirstName = "firstName";
         String expectedLastName = "lastName";
         String expectedGmail = "gmail";
         String expectedSlack = "slack";
         String expectedSkype = "skype";
 
-
-        User user = new User("uuid", "firstName", "lastName", "email", "gmail",
+        UUID uuid = new UUID(1L,2L);
+        User user = new User(uuid, "firstName", "lastName", "email", "gmail",
```

Comment:
if you use the variable only once you can reduce it to  `User user = new User(new UUID(1L, 2L), "firstName", ...`